compatibility_lib/compatibility_lib/get_compatibility_data.py

```diff
@@ -88,14 +74,6 @@ def write_to_status_table():
         res_list = _result_dict_to_compatibility_result(results, py_version)
         store.save_compatibility_statuses(res_list)
 
-        # For github head versions
-        pkg_sets = _generate_pairs_for_github_head()
-        results = checker.get_pairwise_compatibility(
-            python_version=py_version,
-            pkg_sets=pkg_sets)
-        res_list = _result_dict_to_compatibility_result(results, py_version)
-        store.save_compatibility_statuses(res_list)
-
```

Comment:
Is the compatibility status no longer saved to the datastore because of the introduced commit number caching?

Reply:
Yep as we will run the check for gh head version to get the most recent check results, we don't need to run it every 24 hours and store into bigquery.

Reply:
Ok cool! Out of curiosity how does the datastore get updated from now on?

Reply:
There is a kokoro job for populating the cache in datastore and update it.